aspnet/data/entity-framework-6.rst

```diff
@@ -0,0 +1,136 @@
+Configuring Entity Framework 6
+===========================================================
+
+By `Paweł Grudzień <https://github.com/pgrudzien12>`_, `Damien Pontifex <https://github.com/DamienPontifex>`_
+
+This article will show you how to use Entity Framework 6 inside an ASP.NET 5 application.
+
+In this article:
+    - `Setup connection strings and dependency injection`_
+    - `Migrate configuration from config to code`_
+    - `Notes on Automatic Migrations`_
+    
+Prerequisites
+-------------
+    
+Before you start, make sure that you compile against full .NET Framework in your project.json as Entity Framework 6 does not support .NET Core. If you need cross platform features you will need to upgrade to Entity Framework 7. 
```

Comment:
Can you include some info on how to do this, maybe even show the frameworks part of the project.json file.